csharp/src/Ice/Properties.cs

```diff
@@ -55,6 +60,11 @@ public Properties(ref string[] args, Properties? defaults = null)
             {
                 _properties[entry.Key] = entry.Value.Clone();
             }
+
+            foreach (string prefix in defaults._optInPrefixes)
+            {
+                _optInPrefixes.Add(prefix);
+            }
```

Comment:
I think we can use AddRange here
```suggestion
            _optInPrefixes.AddRange(defaults._optInPrefixes);
```